pypeapp/lib/log.py

```diff
@@ -44,10 +37,9 @@
 
 
 PYPE_DEBUG = int(os.getenv("PYPE_DEBUG", "0"))
-
-if _mongo_logging:
-    from bson.objectid import ObjectId
-    MONGO_PROCESS_ID = ObjectId()
+DATABASE = "pypeLogs"
```

Comment:
I would change that to "pype". It does not make sence to have database only for logs.

Reply:
Done